scripts/test-upgrade.sh

```diff
@@ -55,16 +79,25 @@ cd ${OA_DIR}/playbooks
 openstack-ansible lxc-containers-destroy.yml --limit repo_all
 openstack-ansible setup-hosts.yml --limit repo_all
 
-# TASK #3
+# TASK #4
 # https://github.com/rcbops/u-suk-dev/issues/392
 # Upgrade openstack-ansible
 pushd ${OA_DIR}
 export I_REALLY_KNOW_WHAT_I_AM_DOING=true
 echo "YES" | ${OA_DIR}/scripts/run-upgrade.sh
 popd
 
-# TASK #4 TODO: This task should be moved somewhere towards the bottom once
-#               we get a more robust script going.
+# TASK #5
+# https://github.com/rcbops/u-suk-dev/issues/393
+# Run deploy.sh with MaaS and don't run the OSA playbooks.
+# Ultimitely, this will run the RPCO playbooks.
+cd ${BASE_DIR}
+export DEPLOY_MAAS=${DEPLOY_MAAS:-'yes'}
+export DEPLOY_OA='no'
+export BOOTSTRAP_ANSIBLE='no'
```

Comment:
deploy.sh doesn't use BOOTSTRAP_ANSIBLE at the moment, is that a work in progress?